src/components/NcActions/NcActions.vue

```diff
@@ -807,6 +807,133 @@ p {
 }
 </style>
 ```
+
+## NcActions children limitations
+
+`<NcActions>` is supposed to be used with direct `<NcAction*>` children.
+Although it works when actions are not direct children but wrapped in custom components, it has limitations:
+- No `inline` prop property, including a single action display;
+- Accessibility issues, including changed keyboard behavior;
+- Invalid HTML.
+
+```
+<template>
+	<table class="actions-limitations-table">
+		<tr>
+			<th style="width: 50%">Non-direct children</th>
+			<th style="width: 50%">Direct NcAction* children</th>
+		</tr>
+
+		<tr>
+			<th colspan="2">This single button is supposed to be rendered as inline action but it is rendered as a menu:</th>
+		</tr>
+		<tr>
+			<td>
+				<NcActions>
+					<NcActionButtonCustomUser />
+				</NcActions>
+			</td>
+			<td>
+				<NcActions>
+					<NcActionButton>
+						<template #icon>
+							<Account :size="20" />
+						</template>
+						Button
+					</NcActionButton>
+				</NcActions>
+			</td>
+		</tr>
+		<tr>
+			<th colspan="2">This NcActions is supposed to have 2 inline buttons but it has none:</th>
+		</tr>
+		<tr>
+			<td>
+				<NcActions :inline="2">
+					<NcActionButtonCustomUser v-for="i in 4" />
+				</NcActions>
+			</td>
+			<td>
+				<NcActions :inline="2">
+					<NcActionButton v-for="i in 4">
+						<template #icon>
+							<Account :size="20" />
+						</template>
+						Button
+					</NcActionButton>
+				</NcActions>
+			</td>
+		</tr>
+		<tr>
+			<th colspan="2">This NcActions is supposed to have a11y role menu and keyboard navigation but it acts like a dialog:</th>
+		</tr>
+		<tr>
+			<td>
+				<NcActions>
+					<NcActionButtonList />
+				</NcActions>
+			</td>
+			<td>
+				<NcActions>
+					<NcActionButton v-for="i in 4">
+						<template #icon>
+							<Account :size="20" />
+						</template>
+						Button
+					</NcActionButton>
+				</NcActions>
+			</td>
+		</tr>
+	</table>
+</template>
+
+<script>
+import Account from 'vue-material-design-icons/Account.vue'
+
+import { Fragment, h, resolveComponent } from 'vue'
+
+export default {
+	components: {
+		Account,
+
+		NcActionButtonCustomUser: {
+			name: 'NcActionButtonCustomUser',
+			components: { Account },
+			render: () => h(resolveComponent('NcActionButton'), {}, { default: () => 'Button', icon: () => h(Account, { props: { size: 20 } })}),
+		},
+
+		NcActionButtonList: {
+			name: 'NcActionButtonList',
+			components: { Account },
+			render: () => h(Fragment, {}, [
+				h(resolveComponent('NcActionButton'), {}, { default: () => 'Button', icon: () => h(Account, { props: { size: 20 } })}),
+				h(resolveComponent('NcActionButton'), {}, { default: () => 'Button', icon: () => h(Account, { props: { size: 20 } })}),
+				h(resolveComponent('NcActionButton'), {}, { default: () => 'Button', icon: () => h(Account, { props: { size: 20 } })}),
+				h(resolveComponent('NcActionButton'), {}, { default: () => 'Button', icon: () => h(Account, { props: { size: 20 } })}),
+			]),
+		}
+	}
```

Comment:
- Correctly pass `props`
- Bring back `DIV` instead of `Fragment` (to emulate more commonly used wrappers with HTML wrapper)
- Provide slot content via render-function, not vnodes 

```suggestion
import { h, resolveComponent } from 'vue'
import Account from 'vue-material-design-icons/Account.vue'

export default {
	components: {
		Account,

		MyUserActionButton: {
			name: 'MyUserActionButton',
			components: { Account },
			render: () => h(resolveComponent('NcActionButton'), null, { default: () => 'Button', icon: () => h('Account', { size: 20 }) }),
		},

		MyActionsList: {
			name: 'MyActionsList',
			components: { Account },
			render: () => h('div', null, {
				default: () => [
					h(resolveComponent('NcActionButton'), null, { default: () => 'Button', icon: () => h('Account', { size: 20 }) }),
					h(resolveComponent('NcActionButton'), null, { default: () => 'Button', icon: () => h('Account', { size: 20 }) }),
					h(resolveComponent('NcActionButton'), null, { default: () => 'Button', icon: () => h('Account', { size: 20 }) }),
					h(resolveComponent('NcActionButton'), null, { default: () => 'Button', icon: () => h('Account', { size: 20 }) }),
				],
			}),
		}
	}
```